tests/integration/logging/monolog3/test_monolog_decoration_basic.php

```diff
@@ -0,0 +1,208 @@
+    [{"name": "Logging/lines/INFO"},                                              [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Logging/lines/NOTICE"},                                            [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Logging/lines/WARNING"},                                           [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransaction/all"},                                            [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransaction/php__FILE__"},                                    [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransactionTotalTime"},                                       [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransactionTotalTime/php__FILE__"},                           [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/api/get_linking_metadata"},                         [16, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/PHP/Monolog/enabled"},                      [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/library/Monolog/detected"},                         [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/LocalDecorating/PHP/enabled"},              [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/Forwarding/PHP/disabled"},                  [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/Metrics/PHP/enabled"},                      [1, "??", "??", "??", "??", "??"]]
+  ]
+]
+*/
+
+
+/*EXPECT_LOG_EVENTS
+null
+*/
+
+require_once(realpath(dirname(__FILE__)) . '/../../../include/config.php');
+require_once(realpath (dirname ( __FILE__ )) . '/../../../include/tap.php');
+require_once(realpath(dirname(__FILE__)) . '/../../../include/monolog.php');
+require_monolog(3);
+
+use Monolog\Logger;
+use Monolog\LogRecord;
+use Monolog\Handler\StreamHandler;
+
+/* create formatter class that echos the interesting data that the log 
+   decoration should have added. */
+class CheckDecorateFormatter implements Monolog\Formatter\FormatterInterface {
+  public function __construct(?string $dateFormat = null) {
+  }
+  public function format(LogRecord $record) {
+    $nrlinking = $record['extra']['NR-LINKING'] ?? 'NR-LINKING DATA MISSING!!!';
+    $result = preg_match("/(NR\-LINKING)\|([\w\d]+)\|([\w\d]+)\|([\w\d]+)\|([\w\d]+)\|([\w\d\%]+\.php)\|/", $nrlinking, $matches);
+    $linkmeta = newrelic_get_linking_metadata();
+
+    tap_equal(7, count($matches), "All NR-LINKING elements present");
+    if (7 == count($matches)) {
+      tap_equal("NR-LINKING", $matches[1], "NR-LINKING present");
+      tap_equal($linkmeta['entity.guid'] ?? '<missing entity.guid>', $matches[2], "entity.guid correct");
+      tap_equal($linkmeta['hostname'] ?? '<missing hostname>', $matches[3], "entity.guid correct");
```

Comment:
```suggestion
      tap_equal($linkmeta['hostname'] ?? '<missing hostname>', $matches[3], "hostname correct");
```
This will of course require an update in `/*EXPECT` section.

Reply:
The above comment is applicable to all 4 implementations of the formatter.

Reply:
Corrected in commit [29a4f1](https://github.com/newrelic/newrelic-php-agent/commit/29a4f1bd458410cf447493328c45d7d37c46f23b) and commit [0d136b44](https://github.com/newrelic/newrelic-php-agent/commit/0d136b44fd117882d3d71f5b32462c5647490033)